lib/cc/cli/init.rb

```diff
@@ -21,7 +21,8 @@ def create_codeclimate_yaml
         config = { "engines" => {} }
         eligible_engines.each do |engine_name, engine_config|
           config["engines"][engine_name] = {
-            "enabled" => true
+            "enabled" => true,
+            "config" => (additional_duplication_options if engine_name == 'duplication'),
```

Comment:
I don't love hardcoding specific engine names here. Rather than doing that, what if we add:

```yaml
duplication:
  # ...
  default_config:
    languages:
      - ruby
```

To `engines.yml` and use that?

That would work for now, and we can revisit when we run into the issue of: "How do we make the `languages` variable, based on what we find when we init?"